.gitignore

```diff
@@ -1,8 +1,25 @@
-vendor
-.terraform
-*.tfstate*
+# MacOS
+.DS_Store
+
+# Local .terraform directories
```

Comment:
Updated to (mostly) match what [GitHub provides](https://github.com/github/gitignore/blob/main/Terraform.gitignore)